### PR TITLE
Add an OWNERS reviewer to a pull request that goes above the auto-approved bundle size

### LINKED ISSUE #5: bundle-size app should add a comment and tag an ONWERS person with instructions on how to approve a PR that increases the bundle size

### PR DESCRIPTION
**Pull request by danielrozenberg**
Fixes #5

### CODE REVIEW

File: bundle-size/app.js [hunk 1]
```diff
@@ -42,6 +42,20 @@ async function getBuildArtifactsFile(github, filename) {
...
+ *
+ * @param {github} github an authorized GitHub API object.
+ * @throws {Error} on any error.
+ * @return {!Array<string>} an array of OWNERS GitHub usernames.
+ */
+async function getOwners(github) {
+  return (await getBuildArtifactsFile(github, 'OWNERS'))
+      .trim()
+      .split('\n')
+      .filter(line => !line.startsWith('#') && !!line);
```
Comment (rsimha) [inline bundle-size/app.js]: Shouldn't `!!line` be checked before `!line.startsWith('#')`?

File: bundle-size/test/fixtures/requested_reviewers.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "users": [
+    {
+      "login": "jridgewell",
```
Comment (rsimha) [inline bundle-size/test/fixtures/requested_reviewers.json]: I assume this the first of multiple reviewers?

File: bundle-size/test/fixtures/requested_reviewers.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      "following_url": "https://api.github.com/users/jridgewell/following{/other_user}",
+      "gists_url": "https://api.github.com/users/jridgewell/gists{/gist_id}",
+      "starred_url": "https://api.github.com/users/jridgewell/starred{/owner}{/repo}",
+      "subscriptions_url": "https://api.github.com/users/jridgewell/subscriptions",
+      "organizations_url": "https://api.github.com/users/jridgewell/orgs",
+      "repos_url": "https://api.github.com/users/jridgewell/repos",
+      "events_url": "https://api.github.com/users/jridgewell/events{/privacy}",
+      "received_events_url": "https://api.github.com/users/jridgewell/received_events",
+      "type": "User",
+      "site_admin": false
```
Comment (rsimha) [inline bundle-size/test/fixtures/requested_reviewers.json]: Are all these fields necessary? Consider dropping them assuming it's possible.

Comment (danielrozenberg) [reply bundle-size/test/fixtures/requested_reviewers.json]: This is just a copy of how a real response would look like. For tests, I'd rather use a complete real fixture than a mock one

File: bundle-size/test/fixtures/requested_reviewers.json (hunk 2 above)
Comment (danielrozenberg) [reply bundle-size/test/fixtures/requested_reviewers.json]: Correct, a PR with multiple reviewers will have multiple `users` objects

File: bundle-size/app.js (hunk 1 above)
Comment (danielrozenberg) [reply bundle-size/app.js]: Done

**Pull request merged**